library/built-in/SDUI.ts

```diff
@@ -160,12 +161,10 @@ app({
             if (ui.highResFix.saveIntermediaryImage) {
                 graph.SaveImage({ images: graph.VAEDecode({ samples: latent, vae }) })
             }
-            latent = graph.LatentUpscale({
-                samples: latent,
-                crop: 'disabled',
-                upscale_method: 'nearest-exact',
-                height: ui.latent.size.height * ui.highResFix.scaleFactor,
-                width: ui.latent.size.width * ui.highResFix.scaleFactor,
+            latent = graph.NNLatentUpscale({
```

Comment:
@rvion I accidentally included this in the commit. Bit it's not a bad change. I discovered the NNLatentUpscale and really like it. But feel free to exclude it if you want.

Reply:
Oh, didn't know about it; Does it require a custom node?

Reply:
Shoot, yeah. So maybe not good for default latent upscale.

https://github.com/Ttl/ComfyUi_NNLatentUpscale